openmetadata-docs/content/deployment/docker/volumes.md

```diff
@@ -56,6 +119,34 @@ Once these changes are done, restart the container via:
 docker compose down && docker compose up -d
 ```
 
+## Advanced Section for Named Volumes
+We can also use the Named Volumes with the Openmetadata Services in the `docker-compoe` file it self.
+Exxample: 
+
+Named Volume for MYSQL
+```commandline
+version: "3.9"
+volumes:
+  dbdata::
+services:
+  mysql:
+    container_name: openmetadata_mysql
+    image: openmetadata/db:0.11.5
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: password
+    expose:
+      - 3306
+    volumes:
+      - dbdata:/var/lib/mysql
+    networks:
+      app_net:
+        ipv4_address: 172.16.240.10
+```
+
+The above configuration defined one data volume named “dbdata”, which is attached to MySQL container and mounted on /var/lib/mysql directory. This is the default directory used by MySQL to store all data files.
+
```

Comment:
Are we not mentioning how to view the Named Volumes inside the docker daemon running on OS X and Windows ?

Reply:
No

Reply:
For this will create a new issue ticket